### PR TITLE
Use different pac utility for Windows #5346

### PR DESCRIPTION
**Pull request by joesis**
A fix for https://github.com/getlantern/lantern/issues/5346. On some 64bit Windows 10, only the pac utility built on 64bit Windows 10 can actually change PAC setting.

Also added a test file which runs cross-platform.

### CODE REVIEW

File: pac.go [hunk 1]
```diff
@@ -27,7 +30,18 @@ var (
 func EnsureHelperToolPresent(path string, prompt string, iconFullPath string) (err error) {
 	mu.Lock()
 	defer mu.Unlock()
-	pacBytes, err := Asset("pac")
+	assertName := "pac"
+	// Load different binaries for 32bit and 64bit Windows respectively.
+	if runtime.GOOS == "windows" {
+		suffix := "_386.exe"
```
Comment (joesis) [inline pac.go]: Also see `pac2go.sh` for the changed file name

**Comment by joesis** [comment]
It embedded the binaries in https://github.com/getlantern/pac-cmd/pull/3 and needs it to be merged first.

`pac2go.sh` needs to be rerun to sign darwin binary, because `codesign` always failed with "segment fault" on my Mac. Maybe I don't have correct certificate.

File: pac.go [hunk 2]
```diff
@@ -27,7 +30,18 @@ var (
...
+		// https://blogs.msdn.microsoft.com/david.wang/2006/03/27/howto-detect-process-bitness/
+		if strings.EqualFold(os.Getenv("PROCESSOR_ARCHITECTURE"), "amd64") ||
+			strings.EqualFold(os.Getenv("PROCESSOR_ARCHITEW6432"), "amd64") {
+			suffix = "_amd64.exe"
```
Comment (oxtoacart) [inline pac.go]: Any reason not to follow the convention established for linux where each binary is in its own folder?

**Comment by oxtoacart** [comment]
This makes me wonder - if pac-cmd is architecture-dependent, I wonder what else might be? Do you think we should start including a 64-bit version of Lantern itself @joesis?

Note - we don't have to answer this question to finish this PR, it was just something I thought of.

**Comment by oxtoacart** [comment]
@joesis I went ahead and signed the darwin binary.

**Comment by oxtoacart** [comment]
Tests pass on Darwin and Windows XP (32 bit) and 10 (64 bit).

Nice work!

File: pac.go (hunk 2 above)
Comment (joesis) [inline pac.go]: You may have already found out, we need to embed both executables in the single directory.

**Comment by joesis** [comment]
Thanks @oxtoacart !

We'd better stick with single Windows installer unless we find some incompatibility on Lantern itself someday. Fewer installers can save users and us a lot.

**Comment by oxtoacart** [comment]
@joesis I agree, thanks.

**Pull request merged**